### DISCUSSION TITLE
Tailwind way of default fonts

### DISCUSSION THREAD

**Discussion by zkwbbr** [tailwindlabs/tailwindcss#1637] [General]
I gather the [docs](https://tailwindcss.com/docs/font-family) recommend using your own class names for customizing fonts. I think this is easy if it's applied in `<body class="font-foo">`.  But if you need to change default fonts for headings e.g., `h1 h2 h3`, you need to write a [plugin](https://tailwindcss.com/docs/adding-base-styles/#using-a-plugin). Is this correct? The plugin seems quite a bit of work vs just vanilla CSS. E.g., Is doing `h1 { font-family... }` frowned upon?

**Comment by adamwathan** [comment]
You can totally do it in vanilla CSS, we have a bunch of documentation around that here:

https://tailwindcss.com/docs/adding-base-styles/#app

**Reply by zkwbbr** [reply]
Got it. I just thought maybe "touching" CSS is a no-no in TW. I guess fonts are one of those I could give a pass. Thanks!

**Reply by adamwathan** [reply]
Definitely not a no-no, we have lots of features specifically designed to work in your custom CSS 👍 

https://tailwindcss.com/docs/functions-and-directives
